CONTRIBUTING.md

```diff
@@ -18,6 +18,9 @@
 * Install the Ballot app on the new site `bench --site test.ballot install-app ballot`
 * [Access the site conveniently](https://frappeframework.com/docs/user/en/tutorial/create-a-site#access-site-in-your-browser)
   from your computer using `bench --site test.ballot add-to-hosts`
+* Add `"ignore_csrf": "y"` to the `sites/common_site_config.json`
```

Comment:
is `"ignore_csrf":"y"` same as `"ignore_csrf":1` ?

Because usually it is suggested to do the latter

Reply:
updated.